tracks.yaml

```diff
@@ -138,7 +138,7 @@ tracks:
       :{release_inc} --os-name fedora
     - git-bloom-generate -y rosrpm --prefix release/:{ros_distro} :{ros_distro} -i
       :{release_inc} --os-name rhel
-    devel_branch: rolling-devel
+    devel_branch: humble-devel
```

Comment:
Looking at https://github.com/ros-visualization/rqt_image_view/branches/all , it doesn't look like this branch was created yet?

Reply:
It should be created now.